features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Environments/DeploymentOnbording.jsx

```diff
@@ -0,0 +1,312 @@
+                                                                        )}
+                                                                    </TextField>
+                                                                </Tooltip>
+                                                            </Grid>
+                                                        </Grid>
+                                                    </CardContent>
+                                                </Box>
+                                            </Card>
+                                        </Grid>
+                                    ))}
+                                </Grid>
+                            </Box>
+                            <Box mt={2}>
+                                <Button
+                                    color='primary'
+                                    className={classes.button}
+                                    display='inline'
+                                    startIcon={<AddIcon />}
+                                    onClick={handleDescriptionOpen}
+                                >
+                                    Add a description
+                                </Button>
+                                <Typography display='inline' className={classes1.textOptional}>
+                                    (optional)
+                                </Typography>
+                                <br />
+                                {DescriptionOpen && (
+                                    <>
+                                        <TextField
+                                            className={classes1.descriptionWidth}
+                                            name='description'
+                                            margin='dense'
+                                            variant='outlined'
+                                            label='Description'
+                                            inputProps={{ maxLength: maxCommentLength }}
+                                            helperText={(
+                                                <FormattedMessage
+                                                    id='Apis.Details.Environments.Environments.revision
+                                                    .description.deploy'
+                                                    defaultMessage='Brief description of the new revision'
+                                                />
+                                            )}
+                                            multiline
+                                            rows={3}
+                                            defaultValue={description === true ? '' : description}
+                                            onBlur={handleChange}
+                                        />
+                                        {/* <Typography className={classes.textCount} align='right'>
```

Comment:
Shall we remove this commented line ?

Reply:
Fixed https://github.com/wso2/carbon-apimgt/pull/10270/commits/ee20a3b9b54f0df2bafd12c6a3e622938b22935e